packages/devtools-utils/src/source-utils.js

```diff
@@ -0,0 +1,365 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
this is a minor nit, but could you drop this module in `devtools-modules`? The rationale, is that these are the modules that are copied over and thus should be kept in sync